### DISCUSSION TITLE
Multi link download on direct link

### DISCUSSION THREAD

**Discussion by thiendt2k1** [anasty17/mirror-leech-telegram-bot#1753] [Q&A]
When i use /m -i 4 https://xxxx.com/files/[Tu%E1%BA%A5n]%20Fim%20c%E1%BB%A7a%20Tu%E1%BA%A5n/Series/The%20Adventures%20of%20Sinbad/Season%201/The.Adventures.of.Sinbad.S01EP01.NTSC.DVDRemux.DD2.0.mkv
https://xxxx.com/files/[Tu%E1%BA%A5n]%20Fim%20c%E1%BB%A7a%20Tu%E1%BA%A5n/Series/The%20Adventures%20of%20Sinbad/Season%201/The.Adventures.of.Sinbad.S01EP02.NTSC.DVDRemux.DD2.0.mkv
https://xxxx.com/files/[Tu%E1%BA%A5n]%20Fim%20c%E1%BB%A7a%20Tu%E1%BA%A5n/Series/The%20Adventures%20of%20Sinbad/Season%201/The.Adventures.of.Sinbad.S01EP03.NTSC.DVDRemux.DD2.0.mkv
https://xxxx.com/files/[Tu%E1%BA%A5n]%20Fim%20c%E1%BB%A7a%20Tu%E1%BA%A5n/Series/The%20Adventures%20of%20Sinbad/Season%201/The.Adventures.of.Sinbad.S01EP04.NTSC.DVDRemux.DD2.0.mkv

Exact like above(addr changed a bit, but other the same), it return help menu, and i cant get anywhere, so how can i use multi link in this case, is there a list of separators or something.
Thanks

**Comment by anasty17** [comment]
-i when each link sent as message
-b which is bulk to download each link separated by new line
